Makefile

```diff
@@ -41,6 +41,8 @@ ifeq ($(NO_DOCKER), 1)
   CGO_ENABLED = 1
 else
   DOCKER_CMD := docker run --rm -e CGO_ENABLED=1 -v "$(PWD)":/go/src/sigs.k8s.io/cluster-api-provider-aws:Z -w /go/src/sigs.k8s.io/cluster-api-provider-aws openshift/origin-release:golang-1.12
+  TEST_ENV := -e KUBEBUILDER_CONTROLPLANE_START_TIMEOUT=10m -e NO_DOCKER=1
```

Comment:
why can't we reuse `DOCKER_CMD` ?

then test:
should just call `$(DOCKER_CMD) hack/ci-test.sh` ?

Reply:
You're right, that is fixed now